chain/default_syncer_test.go

```diff
@@ -746,8 +747,8 @@ func TestHeaviestIsWidenedAncestor(t *testing.T) {
 
 	// Assert that widened chain is the new head
 	wideTs := testhelpers.RequireNewTipSet(require, link2blk1, link2blk2, link2blk3, forklink2blk1, forklink2blk2, forklink2blk3, forklink2blk4)
-	assertTsAdded(assert, chain, wideTs)
-	assertHead(assert, chain, wideTs)
+	assertTsAdded(assert, chainStore, wideTs)
+	assertHead(assert, chainStore, wideTs)
 }
 
 /* Tests with Unmocked state */
```

Comment:
is the comment below still accurate? seems like weird to leave it in a mysterious state

Reply:
I don't know; it's weird to me too and I haven't gotten around to asking about it/looking into it.